app/app.js

```diff
@@ -0,0 +1,36 @@
+const express = require("express");
+const { getAllTopics } = require("./app.controller");
+
+const app = express();
+app.use(express.json());
+
+//End points
+app.get("/api/topics", getAllTopics);
+
+//Error-handling
+
+app.use((req, res, next) => {
+  res.status(404).send({ msg: "Invalid endpoint" });
+});
+
+app.use((err, req, res, next) => {
+  if (err.status && err.msg) {
+    res.status(err.status).send({ msg: err.msg });
+  } else {
+    next(err);
+  }
+});
+
+app.use((err, req, res, next) => {
+  if (err.code === "22P02") {
+    res.status(400).send({ msg: "Invalid input" });
+  } else {
+    next(err);
+  }
+});
```

Comment:
again I would avoid overcooking your error handling at this stage. It might just need undoing / heavy editing later. If you can't test for error throwing behaviour for any given task, I would not add boiler-plate error handling code right now! see my comments for details.  So probably best to remove until you are able to test these when testing for correct error handling for future endpoints!